hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/FeeCalculatorImpl.java

```diff
@@ -101,7 +103,10 @@ public FeeCalculatorImpl(
         // not something we really should know about here. But, since we're going to replace the fee system later
         // with a simpler model, for now, we'll go ahead and check the transaction body type here.
         final var baseMeta = new BaseTransactionMeta(
-                txBody.memo().getBytes(StandardCharsets.UTF_8).length, // Has to be a faster way...
+                // For some reason in mono-service while auto-creating we don't consider memo bytes for fees
```

Comment:
Probably worth creating a ticket for after differential testing to change this

Reply:
Agreed will do.

Reply:
Done https://github.com/hashgraph/hedera-services/issues/9749